src/bootloader/bootloader.c

```diff
@@ -358,33 +358,44 @@ static void _render_progress(float progress)
 
 static void _render_hash(const char* title, const uint8_t* hash)
 {
-    uint8_t seconds = 10;
-    char message[16];
+    const uint8_t seconds = 10; // how many seconds to show screen
+    const UG_S16 title_margin = 7; // Margin between title and hash
+    const UG_FONT* f_mono = &font_monogram_5X9; // monospaced font
+    const UG_FONT* f_regular = &font_font_a_9X9; // regular font
+
+    char title_buf[50] = {0};
```

Comment:
There's really only 16 chars to use. The rest is filled with the hashes anyway. Keep it at 16? or actually 17 to account for `\0`.

Reply:
I got rid of the buffer altogether, PTAL.